xgi/convert.py

```diff
@@ -302,3 +306,103 @@ def to_incidence_matrix(H, sparse=True, index=False):
     from_incidence_matrix
     """
     return xgi.incidence_matrix(H, sparse=sparse, index=index)
+
+
+def from_bipartite_network(G, create_using=None, dual=False):
```

Comment:
I've always found the use of `create_using` awkward and mostly unnecessary.

If I'm not mistaken, in this instance you can pass an entire hypergraph as `create_using` and this function will add the nodes/edges from the bipartite graph on top of that hypergraph, is that correct?

Reply:
That's correct. In addition, it allows the user to be able to use different constructors (simplicial complex for example).

Reply:
Ok. Let's keep it for now, though I kinda hate the name `create_using`....